Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -400,6 +425,18 @@ def get_last_mirrored_in_time(args):
     return last_mirrored_in_timestamp
 
 
+def get_multiple_incidents_extra_data_response(client, args) -> bool:
+    try:
+        raw_incident: dict = client.get_multiple_incidents_extra_data(args.get('incident_id')) or {}
+        use_get_incident_extra_data = int(raw_incident.get('incident', {}).get('replay', {}).get('alert_count')) > \
```

Comment:
let's make the code here more clear, separate the lines, and store the values in vars.